frontend/javascripts/admin/time/time_line_view.js

```diff
@@ -109,9 +131,57 @@ class TimeLineView extends React.PureComponent<*, State> {
     this.fetchTimeTrackingData();
   };
 
+  getTooltipForEntry(taskId: string, start: Date, end: Date) {
+    const isSameDay = start.getUTCDate() === end.getUTCDate();
+    const duration = end - start;
+    const durationAsString = formatDurationToHoursAndMinutes(duration);
+    const dayFormatForMomentJs = "MMM, DD, YYYY";
```

Comment:
Why this format? I think DD, MMM, YYYY would be the more reasonable default, since our main clients are not in the US.